.github/workflows/parachain.yml

```diff
@@ -80,7 +80,6 @@ jobs:
           --manifest-path parachain/Cargo.toml
           --workspace
           --features runtime-benchmarks
-          --exclude snowbridge-query-events
```

Comment:
Since we're clean the workflow here, would suggest merge `Tests for everything except the beacon light client` with `Benchmark tests for beacon light client`(Something simple as `cargo test --features runtime-benchmarks` should be fine).

Reply:
@yrong can you remember why we split the tests up in the first place?

Reply:
Can't remember correctly. But I would assume since feature `beacon-spec-mainnet` is already covered in `runtime-benchmarks` as in https://github.com/Snowfork/snowbridge/blob/79dba80857f06c550560cbcf818152e0a97137b3/parachain/pallets/ethereum-beacon-client/Cargo.toml#L75, a split in CI is unnessessary.

Reply:
I would rather we focus this PR on the sovereign account issue, and leave CI/devops stuff for a separate PR.

Reply:
The 1-liner change to remove the obsolete `snowbridge-query-events` thingy is reasonable though. 👍

Reply:
> yrong can you remember why we split the tests up in the first place?

I was the one that did that. Because of the following constraints with the beacon client pallet:

* Benchmark tests require the `beacon-spec-mainnet` feature: https://github.com/Snowfork/snowbridge/blob/167b2502d7ae83059cadd78fce0f9b22ae127805/parachain/pallets/ethereum-beacon-client/Cargo.toml#L75

* However, when the `beacon-spec-mainnet` feature is enabled, normal tests are disabled, because they are designed to only work with the minimal spec: https://github.com/Snowfork/snowbridge/blob/167b2502d7ae83059cadd78fce0f9b22ae127805/parachain/pallets/ethereum-beacon-client/src/lib.rs#L15

This is why we need to run the tests separately.

^ @yrong

Reply:
@vgeddes Yes, it's reasonable for the seperation between  `mainnet-spec` and `minimal-spec` for package `snowbridge-ethereum-beacon-client`.  

But the seperation I refer to above is different for running benchmark tests between `ethereum-beacon-client` and other packages which seems somehow unnessessary.

But anyway, we can leave CI/devops stuff for a separate PR.